core/src/main/java/io/opentelemetry/android/SdkPreconfiguredRumBuilder.kt

```diff
@@ -50,6 +51,14 @@ class SdkPreconfiguredRumBuilder
             // might turn off/on additional telemetry depending on whether the app is active or not
             appLifecycleService.registerListener(timeoutHandler)
 
+            val eventLogger =
+                SdkEventLoggerProvider.create(sdk.logsBridge)
+                    .get(OpenTelemetryRum::class.java.simpleName)
+
+            sessionManager.addObserver(SessionIdEventSender(eventLogger))
+            // After addObserver(), we call getSessionId() to trigger a session.start event
+            sessionManager.getSessionId()
```

Comment:
should the `sessionManager` have a `start` method instead for this use case so it's semantically correct?